app/api/slug-exists/route.ts

```diff
@@ -0,0 +1,20 @@
+import { db } from "@/db/client";
+
+export async function POST(request: Request) {
```

Comment:
Typically if an api route doesn't alter any data in the backend (if it's just the R in CRUD ***and it isn't private) I would expect it to be a GET request instead of POST

Reply:
That makes sense. I'll update that, thanks.